main.go

```diff
@@ -67,6 +81,44 @@ func main() {
 				Usage:    "Send logging output to a file instead of STDERR. Unless --log-level is also specified, the default log level changed to WARN.",
 				Action:   actionSetLogFile,
 			},
+			&cli.StringFlag{
+				Category:    "Profiling",
+				Name:        "profile-prefix",
+				EnvVars:     []string{envVarProfilePrefix},
+				Usage:       "Path prefix for profiling data",
+				Value:       "",
+				Destination: &profilePrefix,
+				Action: func(ctx *cli.Context, s string) error {
+					// Set the env var so child processes get it
+					os.Setenv(envVarProfilePrefix, s)
+					return nil
+				},
+				Hidden: true,
+			},
+			&cli.BoolFlag{
+				Category: "Profiling",
+				Name:     "cpu-profiling",
+				EnvVars:  []string{envVarCPUProfiling},
+				Usage:    "Enable the CPU profiler. Profiles are recorded to file \"orchestrion-cpu-${pid}.pprof\"",
+				Action:   actionCPUProfiling,
+				Hidden:   true,
+			},
+			&cli.BoolFlag{
+				Category: "Profiling",
+				Name:     "heap-profiling",
+				EnvVars:  []string{envVarHeapProfiling},
+				Usage:    "Enable the heap profiler. Profiles are recorded to file \"orchestrion-heap-${pid}.pprof\"",
+				Action:   actionHeapProfiling,
+				Hidden:   true,
+			},
+			&cli.BoolFlag{
+				Category: "Profiling",
+				Name:     "execution-tracing",
+				EnvVars:  []string{envVarExecutionTracing},
+				Usage:    "Enable the execution tracer. Traces are recorded to file \"orchestrion-${pid}.trace\"",
+				Action:   actionExecutionTracing,
+				Hidden:   true,
+			},
```

Comment:
These feel to me like they could be a list argument (e.g: `-profile=heap -profile=cpu -profile=execution-tracing`)?

Reply:
Good call, done.